base/src/Data/Macaw/Types.hs

```diff
@@ -23,6 +21,7 @@ The type of machine words, including bit vectors and floating point
 {-# LANGUAGE TypeOperators #-}
 {-# LANGUAGE TypeSynonymInstances #-}
 {-# LANGUAGE UndecidableInstances #-}
+
```

Comment:
Does `hlint` really enforce having a space between the language extensions and the `module` line? This strikes me as odd, since there is no space between the two in other modules (e.g., `Data.Macaw.Discovery.State`), and yet `hlint` doesn't complain about those.

I don't really have a strong opinion one way or the other, but I do find the situation strange. (Similar comments apply to other modules in this PR where a space has been inserted.)

Reply:
Oops, these are just compulsive by me, will revert.